### PR TITLE
Clear token when not needed anymore

### PR DESCRIPTION
**Pull request by emma-miler**
Also blocks migrateme

### CODE REVIEW

**Comment by Erlite** [comment]
based

File: NorthstarDLL/clientauthhooks.cpp [hunk 1]
```diff
@@ -15,6 +15,11 @@ const int NOT_DECIDED_TO_SEND_TOKEN = 0;
 const int AGREED_TO_SEND_TOKEN = 1;
 const int DISAGREED_TO_SEND_TOKEN = 2;
 
+typedef char* (*Auth3PTokenType)();
+Auth3PTokenType Auth3PToken;
+
+char* token_location = 0x0;
```
Comment (pg9182) [inline NorthstarDLL/clientauthhooks.cpp:21]: Would make this static, but it's not urgent.

**Review by pg9182 — APPROVED** [review]
LGTM

File: NorthstarDLL/clientauthhooks.cpp (hunk 1 above)
Comment (Erlite) [reply NorthstarDLL/clientauthhooks.cpp:21]: unbased

**Pull request merged**